lib/customer.rb

```diff
@@ -0,0 +1,46 @@
+require "awesome_print"
+require "csv"
+
+class Customer
+  attr_reader :id
+  attr_accessor :email, :address
+
+  def initialize(id, email, address)
+    @id = id
+    @email = email
+    @address = address
+  end
+
+  def self.all
+    customers = []
+    CSV.open("data/customers.csv", "r").each do |instance|
+      instance
```

Comment:
When you run `rake`, Minitest gives a warning, saying that this line (that just has `instance`) is probably not doing anything important, and can be deleted. I agree!